src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -262,6 +262,7 @@ public function endTest($test, $time)
         if ($this->runsInSeparateProcess) {
             $deprecations = file_get_contents($this->runsInSeparateProcess);
             unlink($this->runsInSeparateProcess);
+            putenv('SYMFONY_DEPRECATIONS_SERIALIZE=');
```

Comment:
the `=` should be removed, to really remove the env var (the `=` makes it the empty string, where we want to delete it, isn't it?)